src/components/Wizard/index.tsx

```diff
@@ -11,7 +11,7 @@ import NFTDetails from "./Presets/NFTDetails";
 import "./Wizard.scss";
 
 interface HeaderProps {
-  header?: string;
+  header?: ReactNode;
```

Comment:
This should be `ReactNode | string`

Reply:
string is the part of ReactNode

Reply:
` type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;`
`type ReactChild = ReactElement | ReactText;`
`type ReactText = string | number;`

Hope you understand it.

Reply:
I just remember that I was going to add close icon in `header`, but not added it yet. I think it should be handle in Wizard

Reply:
I can see there are some places you used ` ReactNode | string`, those should be `ReactNode`

Reply:
Oh I see - makes sense I didn't know that! Thanks 🚀